pulp_container/app/registry_api.py

```diff
@@ -1219,17 +1219,17 @@ def put(self, request, path, pk=None):
 
             manifests_to_list = []
             for manifest in found_manifests:
-                platform = manifests[manifest.digest]
                 manifest_to_list = models.ManifestListManifest(
                     manifest_list=manifest,
                     image_manifest=manifest_list,
-                    architecture=platform["architecture"],
-                    os=platform["os"],
-                    features=platform.get("features", ""),
-                    variant=platform.get("variant", ""),
-                    os_version=platform.get("os.version", ""),
-                    os_features=platform.get("os.features", ""),
                 )
+                if platform := manifests[manifest.digest]:
+                    manifest_to_list.architecture = (platform["architecture"],)
+                    manifest_to_list.os = (platform["os"],)
+                    manifest_to_list.features = (platform.get("features", ""),)
+                    manifest_to_list.variant = (platform.get("variant", ""),)
+                    manifest_to_list.os_version = (platform.get("os.version", ""),)
+                    manifest_to_list.os_features = (platform.get("os.features", ""),)
```

Comment:
I do not understand the purpose of the encapsulating brackets here. What is it needed for? This does not look good to me.
```
>>> platform = {"os": "blabla", "architecture": "amd"}
>>> (platform.get("architecture"),)
('amd',)
```
```
In [16]: ManifestListManifest.objects.filter().update(os=("amd",))
Out[16]: 1

In [17]: ManifestListManifest.objects.get().os
Out[17]: "('amd',)"
```